packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -106,7 +109,41 @@ export const CreateAlertDefinition = () => {
       }
     }
   });
+  const { data: notificationData } = useNotificationChannels();
+  const notificationChannelWatcher = useWatch({ control, name: 'channel_ids' });
 
+  const onChangeNotifications = (notifications: NotificationChannel[]) => {
+    const notificationTemplateList = notifications.map(
+      (notification) => notification.id
+    );
+    setValue('channel_ids', notificationTemplateList);
+  };
+  const [openAddNotification, setOpenAddNotification] = React.useState(false);
+
+  const onSubmitAddNotification = (notificationId: number) => {
+    setValue('channel_ids', [...notificationChannelWatcher, notificationId], {
+      shouldDirty: false,
+      shouldTouch: false,
+      shouldValidate: false,
+    });
+    setOpenAddNotification(false);
+  };
+
+  const getNotifications = React.useMemo(() => {
+    return (
+      notificationData?.data.filter(
+        (notification) => !notificationChannelWatcher.includes(notification.id)
+      ) ?? []
+    );
+  }, [notificationChannelWatcher, notificationData]);
+
+  const getSelectedNotifications = React.useMemo(() => {
```

Comment:
can we not name variables like getNotification, it gives the idea that these are methods but they are just list or normal variables

Reply:
Changed getNotifications to notifications and getSelectedNotifications to selectedNotifications